src/types/index.ts

```diff
@@ -173,6 +173,30 @@ export type PaymentRequestButtonElementComponent = React.FC<
   PaymentRequestButtonElementProps
 >;
 
+export interface AuBankAccountElementProps extends ElementProps {
+  /**
+   * An object containing [Element configuration options](https://stripe.com/docs/js/elements_object/create_element?type=paymentRequestButton).
+   */
+  options?: stripeJs.StripeAuBankAccountElementOptions;
+
+  /**
+   * Triggered when data exposed by this Element is changed (e.g., when there is an error).
+   * For more information, refer to the [Stripe.js reference](https://stripe.com/docs/js/element/events/on_change?type=ibanElement).
```

Comment:
`type=ibanElement` should probably be `type=auBankAccountElement`

Reply:
fixed!